examples/dbolya-yolact/README.md

```diff
@@ -42,7 +42,7 @@ run the following examples.
 
 | Model Name     |      Stub      | Description |
 |----------|-------------|-------------|
-| yolact-base |  | Dense full precision YOLACT model with darknet53 backbone|
+| yolact-base |  | Dense full precision YOLACT model with a DarkNet-53 backbone|
```

Comment:
@markurtz or @rahul-tuli should this have a stub format example in the blank column?

Reply:
Will update this as soon as model is uploaded to SparseZoo

Reply:
updated with the proper stubs and table. Going to land this as and we can open up another PR if there are additional issues